src/main/java/com/google/sps/servlets/SessionInit.java

```diff
@@ -0,0 +1,8 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.*;
```

Comment:
Using a wildcard in imports can cause issues if the package contains a Class that interferes with other classes your using.  

https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad